regulations/static/regulations/css/less/module/paragraph-markers.less

```diff
@@ -59,5 +60,16 @@ ol {
 }
 
 .level-2 > li > p {
-    margin-top: 0;
+    margin-top: 1em;
+}
+
+// move key term headings up to center with paragraph markers
+.paragraph-marker + .key-term {
+  float: left;
+  margin-top: -5px;
```

Comment:
Those properties actually cascade down to other levels, just not `level-1` so that is why I apply the the properties to all key terms and then make a rule for `level-2` (also cascades to lower levels) as well.